docs/knowledge-base/posts/routing-l2-networks.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Routing Layer 2 Networks with VergeOS
+slug: routing-l2-networks
+description: Describes an overview and specific scenario in which physical devices are routed via Layer 3 Verge networks.
+draft: false
+date: 2025-04-09T18:42:31.354Z
+tags:
+  - network
+  - networking
+  - network blocks
+  - subnet
+  - virtual wire
+  - vwire
+  - layer2
+  - l2
+categories:
+  - Network
+editor: markdown
+dateCreated: 2025-04-09T11:15:31.354Z
+---
+
+## Routing IP Traffic to L2 physical networks
+This article will walk through an efficient method of routing layer 3 IP traffic for layer 2 physical networks with the powerful VergeOS networking system. There are multiple ways to achieve this, however the objective of this article is to provide clear and concise guidance on a specific common scenario. This article should be particularly of use and interest to engineers and administrators of Edge deployments in which physical devices like phones, cctv cameras, or network equipment require communication with workloads in VergeOS that may be running on different IP subnets. Using this method, an operator can achieve direct local communication between those physical devices outside of VergeOS and the internal VergeOS managed networks.
+    
+### Steps Overview
+1. [Create Layer2 external Network](/knowledge-base/network-blocks/)
```

Comment:
This link does not tell me how to create an external network, /knowledge-base/create-external-network/ may be more relevant